src/tyr/actor.cc

```diff
@@ -69,7 +69,7 @@ namespace valhalla {
       skadi::skadi_worker_t skadi_worker;
     };
 
-    actor_t::actor_t(const boost::property_tree::ptree& config): pimpl(new pimpl_t(config)) {
+    actor_t::actor_t(const boost::property_tree::ptree& config, bool auto_cleanup): pimpl(new pimpl_t(config)), auto_cleanup(auto_cleanup) {
```

Comment:
i changed the actor constructor to take a flag that lets you say you want it to take care of cleaning up between actions. in a single threaded application this is convenient. in an application where you want to hand back a response before you waste time cleaning up you can choose not to make use of it